xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/dto/account/OkCoinFuturesInfoFixed.java

```diff
@@ -0,0 +1,65 @@
+package org.knowm.xchange.okcoin.dto.account;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class OkCoinFuturesInfoFixed {
+  private final OkcoinFuturesFundsFixed btcFunds;
+  private final OkcoinFuturesFundsFixed ltcFunds;
+  private final OkcoinFuturesFundsFixed bchFunds;
+  private final OkcoinFuturesFundsFixed ethFunds;
+  private final OkcoinFuturesFundsFixed etcFunds;
+  private final OkcoinFuturesFundsFixed xrpFunds;
+  private final OkcoinFuturesFundsFixed eosFunds;
+  private final OkcoinFuturesFundsFixed btgFunds;
+
+  public OkCoinFuturesInfoFixed(
+      @JsonProperty("btc") final OkcoinFuturesFundsFixed btcFunds,
```

Comment:
looking at the [api documentation](https://github.com/okcoin-okex/API-docs-OKEx.com/blob/master/API-For-Futures-EN/REST%20API%20for%20FUTURES.md), we get such a structure:

```
{  
   "info":{  
      "btc":{  
         "balance":99.95468925,
         "contracts":[  
            {  
               "available":99.95468925,
               "balance":0.03779061,
               "bond":0,
               "contract_id":20140815012,
               "contratct_type":"this_week",
               "freeze":0,
               "profit":-0.03779061,
               "unprofit":0
            }
         ],
         "rights":99.95468925
      },
      "ltc":{  
         "balance":77,
         "contracts":[  
            {  
               "available":99.95468925,
               "balance":0.03779061,
               "bond":0,
               "contract_id":20140815012,
               "contract_type":"this_week",
               "freeze":0,
               "profit":-0.03779061,
               "unprofit":0
            }
         ],
         "rights":100
      }
   },
   "result":true
}
```
you should try to create a generic code with a hash map where the key is the currency
the structure, you provided here, will need to be changed every time a new currency is listed on the exchange. you do not want it this way